it/controllers/CheckYourAnswersControllerISpec.scala

```diff
@@ -556,7 +560,9 @@ class CheckYourAnswersControllerISpec extends IntegrationSpecCommonBase {
       SessionKeys.startDateOfPeriod -> LocalDate.parse("2020-01-01"),
       SessionKeys.endDateOfPeriod -> LocalDate.parse("2020-01-01"),
       SessionKeys.dueDateOfPeriod -> LocalDate.parse("2020-02-07"),
-      SessionKeys.dateCommunicationSent -> LocalDate.parse("2020-02-08"),
+      SessionKeys.firstPenaltyCommunicationDate -> LocalDate.now().plusDays(1),
+      SessionKeys.secondPenaltyCommunicationDate -> LocalDate.now().plusDays(1),
+      SessionKeys.dateCommunicationSent -> LocalDate.now().plusDays(1),
```

Comment:
Do you need dateCommunicationSent?

Reply:
Yes as otherwise it fails on DataRetrieval